src/plugins/vis_types/timeseries/public/application/components/use_index_patter_mode_callout.tsx

```diff
@@ -33,22 +33,22 @@ export const UseIndexPatternModeCallout = () => {
     <EuiCallOut
       title={
         <FormattedMessage
-          id="visTypeTimeseries.visEditorVisualization.indexPatternMode.notificationTitle"
-          defaultMessage="TSVB now supports index patterns"
+          id="visTypeTimeseries.visEditorVisualization.dataViewMode.notificationTitle"
+          defaultMessage="TSVB now supports data views"
```

Comment:
Open question: Do we still need this callout?

I'm not familiar with the history so I'm genuinely unsure.

Reply:
This callout appears only when the users select the string mode (which supports dataviews and es indices). The string mode is not recommended by us so we want to keep it (we will slightly change the text to not include the `now`) in order to notify the users about the data view mode (and the link to our docs). Unfortunately, we can't do it automatically so we want to use this callout to advertise it and convince them to do this change on their own.

Reply:
Thanks for the extra context @stratoula. I'll mark this as resolved then.